core/src/com/nukesz/game/GameScreen.java

```diff
@@ -89,6 +90,10 @@ public void resize(int width, int height){
     @Override
     public void render(float delta) {
         switch (state) {
+
```

Comment:
Formatter should clean it up. If not please share your formatter settings with me.